Client/Frontend/AuthenticationManager/BasePasscodeViewController.swift

```diff
@@ -39,7 +39,11 @@ class BasePasscodeViewController: UIViewController {
         super.viewDidLoad()
         view.backgroundColor = SettingsUX.TableViewHeaderBackgroundColor
         updateRightBarButtonItem()
-        automaticallyAdjustsScrollViewInsets = false
+        if #available(iOS 11.0, *) {
+            scrollView?.contentInsetAdjustmentBehavior = .never
```

Comment:
This scroll view is never created or used anywhere, so setting this does nothing. I don't even think `automaticallyAdjustsScrollViewInsets` is used in this controller since there's no scroll view on it. You should instead add `contentInsetAdjustmentBehavior = .never` to its subclasses that create `UIScrollView`'s